### PR TITLE
Add the requirements for the Millstone-2

### PR DESCRIPTION
**Pull request by GzimAsani**
Finished the Millstone-2 and apply for a review

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
+source 'https://rubygems.org'
+  .gem 'rubocop', '~>0.81.0'
```
Comment (peterrobert) [inline Gemfile]: Kindly remove the period in front of the  gem word

**Review by peterrobert — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED :recycle: :stop_button: 
### HELLO :wave:  @GzimAsani AND @CornerstoneII VERY NICE PROGRESS SO FAR YOU HAVE DONE SOME GOOD JOB. JUST A FEW CHANGES TO BE DONE. KINDLY IMPROVE ON THE FOLLOWING:

- [x] There are some rubocop offenses. Kindly fix all the offenses
some few offences rub

- [x] Resolve all the conflicts that are in this branch

**also improve on the following:**

- [x] The game tells which player turn it is (without computing who the winner is). You don't have to write the logic. **[required]**

- [x] The game should ask a player to select from available moves. **[required]**

- [x] The game informs player if the selected move is invalid. **[required]**

- [x] The game displays board after the player move. **[required]**

* [x] game informs player if the selected move is a winning move 

* [x] `[REQUIRED]` game informs player if the selected move is a draw move

* [x] `[REQUIRED]` game repeats all actions for the next player's move

**if you have any question, kindly reach me on slac @peterrobert**

### HAPPY CODING

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello Oluwaseun Iyadi & Gzim Asani** 
Nice job with the improvements to your project :muscle:  :muscle: 
We still need to fix some issues:

- [x] Game board should be displayed
- [x] Game board should be updated when a player makes a valid movement
- [x] Game should inform the player if the selected move is a draw move
- [x] game should inform the player if the selected move is a winning move


Please request another review once you are ready! :+1: 

Happy coding :relaxed:

Oscar Andrés Russi Porras

**Review by ghost — CHANGES_REQUESTED** [review]
# Status: Changes Required

### Hello team,

**Great job on your project. However, there are a few fixes needed before the project can be approved.**

___
### :spades: Mandatory Changes :spades: 
___
** Basic Hint
image
___

- Please add a scenario in the code where the game displays board after the player move.

- Well, the game contains two possible results. Either there is a winner or it is a draw. I can see puts that describe both situations. You can also display the game board in the code about this scenario.
- The main purpose of the milestone is that you have all the possible scenarios on your hand before you start implementing the logic. So a loop is actually needed to describe that you are aware that are some repetitive tasks gonna be placed in here.
```ruby
game_on = true

# initial instructions for player

while game_on
  # loop for each move
  
  if winner # and/or draw 
    game_on = false
  end
end
```
___

### You can use this [approach](https://repl.it/join/rqbjyzum-blackpintz) as a reference on how to go about it.
___
## :herb: Happy Coding :herb: 

**PS: You can reach me on slack (Rose Wanjohi).**

**Review by elmejdki — CHANGES_REQUESTED** [review]
# status: require changes :recycle: 
## Hi, I noticed that you have skipped the required changes by the previous TSE's

please make all the required changes by the previous TSE otherwise your project will not be approved.

## Happy Coding :tophat:

**Review by kevcap — CHANGES_REQUESTED** [review]
# [STATUS] CHANGES REQUIRED♻️

#### Hey team,  your project is looking really good, however, to accomplish as the actual project, you're required to do these changes, listed below:
 
## [REQUIRED CHANGES]
- [ ] Solve the conflict in bin/main.rb
- [ ] Enable your Linter!
- [ ] Your game tells which player turn it is (without computing who the winner is)
- [ ] game informs player if the selected move is a draw move (without computing draw)
- [ ] game informs player if the selected move is a winning move (without computing the winning move)

## Please make the required changes and resubmit for a new code review once you are ready💪

## As always, I wish you all the very best and see you both at the top🚀


👨🏻‍💻 **TSE: Kevin Alves**

  

📫 Have any question? Better call Kevin:

<a  href="mailto:kevinalves.js@gmail.com"> gmail</a> [Twitter Follow](https://twitter.com/intent/follow?screen_name=kevinnnmn) [](https://microverse-students.slack.com/team/UT86D3AP4)

<sub>*We're here to help you, to ensure that you're using the best practices and getting you ready for the job market. In real-world, you'll have a lot of changes requests for your real clients, it's good to get used to it.</sub>

**Comment by CornerstoneII** [comment]
> # status: require changes recycle
> ## Hi, I noticed that you have skipped the required changes by the previous TSE's
> 
> please make all the required changes by the previous TSE otherwise your project will not be approved.
> ## Happy Coding tophat
@elmejdki Next time when you make a review try to explain clearly what exactly is to be done, your vague reviews are stressful to understand.

**Pull request closed without merging**